src/tts/tts_pytorch.py

```diff
@@ -208,8 +221,8 @@ def make_batchset(data, batch_size, max_length_in, max_length_out,
         logging.info('# utts: ' + str(len(sorted_data)))
         # change batchsize depending on the input and output length
         while True:
-            ilen = int(sorted_data[start][1]['input'][0]['shape'][0])
-            olen = int(sorted_data[start][1]['output'][0]['shape'][0])
+            ilen = int(sorted_data[start][1]['output'][0]['shape'][0])
```

Comment:
Is it a bug? Why did you change it?

Reply:
Because we use same json file for tts recipe, the input and output are reversed.

Reply:
In the case of ljspeech, we use shuffuled minibatch, therefore there is no problem.